MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,13 +20,19 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onLoadingError: Observer<String>?
 
 	func loadFeed() {
 		onLoadingStateChange?(true)
 		feedLoader.load { [weak self] result in
-			if let feed = try? result.get() {
+			switch result {
+			case let .success(feed):
 				self?.onFeedLoad?(feed)
+
+			case let .failure(error):
```

Comment:
Please remove `error`, since it's not used. Otherwise it generates a warning, and according to the challenge guidelines:

> 4. The project should build without warnings.

```suggestion
			case let .failure:
```